projects/core/src/cms/facade/cms.service.ts

```diff
@@ -278,4 +280,33 @@ export class CmsService {
   setPageFailIndex(pageContext: PageContext, value: string): void {
     this.store.dispatch(new CmsActions.CmsSetPageFailIndex(pageContext, value));
   }
+
+  /**
+   * Get routerLink from CMS data
+   * @param data CMSComponent data
+   */
+  getRouterLink(data: any): Observable<any> {
```

Comment:
better not add this function here. This service is for `CMS`, but not for some specific cms component.